### PR TITLE
added content that describes the system project

### PR DESCRIPTION
**Pull request by MBishop17**
@deniseschannon , I noticed the section about projects didn't have anything about the default project, so I added it now.

@kinarashah , don't worry about any of the stuff I added about the default project. Just review the part about system projects.

### CODE REVIEW

File: content/rancher/v2.x/en/cluster-provisioning/rke-clusters/options/_index.md [hunk 1]
```diff
@@ -21,8 +21,8 @@ From this section you can choose:
...
 
 - The [Network Provider](https://kubernetes.io/docs/concepts/cluster-administration/networking/) that the cluster uses. Out of the box, Rancher supports:
-
-    - **[Canal](https://github.com/projectcalico/canal)**: In v2.0.0 - v2.0.4 and v2.0.6, this was the default option for these clusters was Canal with network isolation. With the network isolation automatically enabled, it prevented any pod communication between [projects]({{< baseurl >}}/rancher/v2.x/en/k8s-in-rancher/projects-and-namespaces/).
+    
+    - <a id="canal"></a>**[Canal](https://github.com/projectcalico/canal)**: In v2.0.0 - v2.0.4 and v2.0.6, this was the default option for these clusters was Canal with network isolation. With the network isolation automatically enabled, it prevented any pod communication between [projects]({{< baseurl >}}/rancher/v2.x/en/k8s-in-rancher/projects-and-namespaces/).
```
Comment (deniseschannon) [inline content/rancher/v2.x/en/cluster-provisioning/rke-clusters/options/_index.md]: Are we going to add in options of what v2.0.7+ is for canal?

Comment (MBishop17) [reply content/rancher/v2.x/en/cluster-provisioning/rke-clusters/options/_index.md]: Yes, it's in PR #576

File: content/rancher/v2.x/en/k8s-in-rancher/projects-and-namespaces/_index.md [hunk 2]
```diff
@@ -9,7 +9,44 @@ aliases:
...
+_available as of v2.0.7_
+
+When troubleshooting, you can view the `system` project to check if important namespaces in the Kubernetes system are working properly. This easily accessible project saves you from troubleshooting individual system namespace containers.
+
+To open it, open the **Global** menu, and then select the `system` project for your cluster. 
+
+The `system` project:
+
+- Is automatically created when you provision a cluster.
+- Lists all namespaces that exist in `v3/settings/system-namespaces`.
```
Comment (kinarashah) [inline content/rancher/v2.x/en/k8s-in-rancher/projects-and-namespaces/_index.md]: not totally sure, but ```Lists all namespaces specified in `v3/settings/system-namespaces` if they exist```

Comment (kinarashah) [inline content/rancher/v2.x/en/k8s-in-rancher/projects-and-namespaces/_index.md]: not totally sure, but ```Lists all namespaces specified in `v3/settings/system-namespaces` if they exist```

**Pull request merged**